sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriter.cs

```diff
@@ -305,4 +310,16 @@ internal static bool ShouldWriteAsJson<T>(IPersistableModel<T> model, ModelReade
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     private static bool IsJsonFormatRequested<T>(IPersistableModel<T> model, ModelReaderWriterOptions options)
         => options.Format == "J" || (options.Format == "W" && model.GetFormatFromOptions(options) == "J");
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    private static ModelReaderWriterOptions GetOptions(ModelReaderWriterOptions? options)
```

Comment:
I was going to mention something about "maybe a struct instead" to avoid the class allocation, but given the context of serialization, doesn't seem like a concern.